src/packages.builds

```diff
@@ -10,7 +10,7 @@
     <Project Include="Native\pkg\**\*.builds">
       <AdditionalProperties>$(AdditionalProperties);SkipCreatePackageOnMissingFiles=true</AdditionalProperties>
     </Project>
-    <Project Include="*\pkg\*.builds">
+    <Project Include="*\pkg\*.builds" Condition="'$(Native)' != 'true'">
```

Comment:
Native is too general a of a property name and will likely clash. Given you are putting this here I would probably call this SkipManagedPackageBuild.